### PR TITLE
Modify CarouselViewController to address glitches

### PR DESCRIPTION
**Pull request by Qata**

### CODE REVIEW

**Comment by barefeettom** [comment]
Thanks, Charlotte 😊

File: BFWControls/Modules/Carousel/Controller/CarouselViewController.swift [hunk 1]
```diff
@@ -44,6 +44,9 @@ class CarouselViewController: UICollectionViewController {
         return cellIdentifiers?.count ?? 0
     }
     
+    var pageControl = UIPageControl()
+    var collectionViewSize: CGSize?
```
Comment (barefeettom) [inline BFWControls/Modules/Carousel/Controller/CarouselViewController.swift:48]: Why does collectionViewSize need to be public?

Comment (Qata) [reply BFWControls/Modules/Carousel/Controller/CarouselViewController.swift:48]: I'm currently using it in my subclass. I'm looking to replace that usage right now.

File: BFWControls/Modules/Carousel/Controller/CarouselViewController.swift [hunk 2]
```diff
@@ -86,17 +83,14 @@ class CarouselViewController: UICollectionViewController {
         return identifiers
     }
     
-    private var looping: Bool {
-        return looped && (cellIdentifiers?.count ?? 0) > 1
+    var looping: Bool {
```
Comment (barefeettom) [inline BFWControls/Modules/Carousel/Controller/CarouselViewController.swift:86]: Why does looping need to be public?

File: BFWControls/Modules/Carousel/Controller/CarouselViewController.swift [hunk 3]
```diff
@@ -121,16 +115,17 @@ class CarouselViewController: UICollectionViewController {
                                                 animated: animated)
     }
     
+    private func updatePageControl() {
+        pageControl.frame.origin.x = (collectionView!.bounds.width - pageControl.frame.width) / 2 + collectionView!.contentOffset.x
+        pageControl.frame.origin.y = collectionView!.bounds.height - pageControl.frame.height + collectionView!.contentOffset.y - controlInsetBottom
+        pageControl.currentPage = loopedPageForPage(page + 1)
```
Comment (barefeettom) [inline BFWControls/Modules/Carousel/Controller/CarouselViewController.swift:121]: Why +1 ?

Comment (Qata) [reply BFWControls/Modules/Carousel/Controller/CarouselViewController.swift:121]: There's a bug that's causing all pages to show as `page  - 1`. I've messaged you about it in Slack.

**Pull request merged**